google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/PushConfig.java

```diff
@@ -1,363 +0,0 @@
-/*
- * Copyright 2016 Google Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.google.cloud.pubsub;
-
-import static com.google.common.base.Preconditions.checkNotNull;
-
-import com.google.common.base.MoreObjects;
-import com.google.common.collect.ImmutableMap;
-
-import java.io.Serializable;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
-
-/**
- * Google Cloud Pub/Sub configuration for a push subscription.
- *
- * <p>In a push subscription, the Pub/Sub server sends a request to the subscriber application. A
- * {@code PushConfig} object can be used to configure the application endpoint. The subscriber's
- * HTTP response serves as an implicit acknowledgement: a success response indicates that the
- * message has been succesfully processed and the Pub/Sub system can delete it from the
- * subscription; a non-success response indicates that the Pub/Sub server should resend it
- * (implicit "nack").
- *
- * @see <a href="https://cloud.google.com/pubsub/subscriber">Subscriber Guide</a>
- */
```

Comment:
I'm not so much referring to this specific link as to all of the doc comments in the file. When we wrote this client, we added documentation that would help people when using the client. I would like to still make the information available somehow, even though we are losing these wrapper classes. That either means putting it into higher-level documentation, or modifying the original proto documentation.